wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+BackupImport.swift

```diff
@@ -23,8 +23,6 @@ extension NSManagedObjectContext {
     /// Prepare a backed up database for being imported, deleting self client, push token etc.
     func prepareToImportBackup() {
         setPersistentStoreMetadata(nil as Data?, key: ZMPersistedClientIdKey)
-        setPersistentStoreMetadata(nil as Data?, key: PersistentMetadataKey.pushToken.rawValue)
-        setPersistentStoreMetadata(nil as Data?, key: PersistentMetadataKey.pushKitToken.rawValue)
```

Comment:
is it unused?

Reply:
We seem to have migrated from storing these values in the persistent store to storing them in the UserDefaults.
See `PushTokenStorage` and `PushTokenService`.
@netbe